src/main/java/hw_3/Task1.java

```diff
@@ -0,0 +1,33 @@
+package hw_3;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+import static java.awt.SystemColor.text;
+public class Task1 {
+    public static void main(String[] args) {
+        // ДЗ рядки
+        //Виведіть у консоль один рядок з числами з масиву (які ми створювали в 1-му завданні), розділеними “;”
+        int[] array = {1, 2, 3, 4, 5};
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < array.length; i++) {
+            result.append(array[i]);
+            if (i < array.length - 1) {
+                result.append(";");
+            }
+        }
+        System.out.println(result.toString());
```

Comment:
You can use just 'result' in System.out.println. without .toString()

Reply:
fixed

Reply:
not